src/operator/nn/cudnn/cudnn_pooling-inl.h

```diff
@@ -81,7 +81,8 @@ class CuDNNPoolingOp {
     CHECK_EQ(s->dnn_handle_ownership_, mshadow::Stream<gpu>::OwnHandle);
     typename DataType<DType>::ScaleType alpha = 1.0f;
     typename DataType<DType>::ScaleType beta = 0.0f;
-    this->Init(s, in_data, out_data);
+    if (!this->Init(s, in_data, out_data))
+      LOG(FATAL) << "cuDNN Pooling invoked with unsupported parameters.";
```

Comment:
e.g. otherwise it's hard to figure out where the problem is with this error message.

Reply:
Thanks for this tip.  Updated per your suggestion.

Reply:
Any other issues you'd like me to look at?